apps/charterafrica/src/components/Articles/Articles.js

```diff
@@ -65,7 +66,14 @@ const Articles = React.forwardRef(function Articles(props, ref) {
   if (filtering && articlesRef.current) {
     articlesRef.current.scrollIntoView({ behavior: "smooth" });
   }
-  const res = useArticles(slug, { locale, q, sort, page, pageSize });
+  const res = useArticles(collection || slug, {
+    locale,
+    q,
```

Comment:
What is this variable q? the other variables represent the actual representation I believe but q is not clear?

Reply:
`q` = `query` e.g. `https://www.google.com/search?q=code+for+africa` or `https://github.com/search?q=code+for+africa&type=repositories`